core/templates/dev/head/pages/collection_editor/collection_editor_navbar_directive.html

```diff
@@ -55,14 +55,14 @@
     </li>
   </ul>
 
-  <ul class="nav navbar-nav oppia-navbar-nav navbar-right">
+  <ul class="nav navbar-nav oppia-navbar-nav navbar-right" ng-if="!isCollectionLoading()">
```

Comment:
So I think my concern with ``isCollectionLoading()`` is whether we need it or, put another way, what happens when it's false? This won't show up; is that the correct behavior that we want?